src/lib/components/Widget.tsx

```diff
@@ -95,18 +107,19 @@ export default function Widget(props: PropsWithChildren<WidgetProps>) {
     provider,
     jsonRpcEndpoint,
     width = 360,
-    dialog,
+    dialog: userDialog,
     className,
     onError,
   } = props
 
-  const [wrapper, setWrapper] = useState<HTMLDivElement | null>(null)
+  const [dialog, setDialog] = useState<HTMLDivElement | null>(null)
   return (
     <StrictMode>
       <I18nProvider locale={locale}>
         <ThemeProvider theme={theme}>
-          <WidgetWrapper width={width} className={className} ref={setWrapper}>
-            <DialogProvider value={dialog || wrapper}>
+          <WidgetWrapper width={width} className={className}>
+            <DialogWrapper ref={setDialog} />
```

Comment:
a little weird to have a `...Wrapper` that isn't wrapping anything

Reply:
[](git.reservetory🌄)